randovania/data/json_data/prime1.txt

```diff
@@ -2015,32 +2093,44 @@ Asset id: 333016381
 > Dock to Transport to Chozo Ruins West; Heals? False
   > Transport to Chozo Ruins West/Dock to Transport Tunnel A
       Trivial
+  > Transport Tunnel A/Dock to Tallon Canyon
+      Trivial
 
 > Dock to Tallon Canyon; Heals? False
   > Tallon Canyon/Dock to Transport Tunnel A
       Trivial
+  > Transport Tunnel A/Dock to Transport to Chozo Ruins West
+      Trivial
 
 ----------------
 Root Tunnel
 Asset id: 1078461657
 > Dock to Root Cave; Heals? False
   > Root Cave/Dock to Root Tunnel
       Trivial
+  > Root Tunnel/Dock to Tallon Canyon
+      Missile
```

Comment:
Same as above. Door data should be handled in the Node info and not in the requirements.